src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -31,4 +31,9 @@ We recommend using the following best practices when developing themes:
    <a href="#"><?= __('Click to download'); ?></a>
    ```
 
+1. Make use of the mobile first approach when inheriting blank or luma themes which has already been followed in these themes.
+1. Mangento have a set of [coding stadrads]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies, Do refer and follow them when needed.
+1. Do not repeat a work, instead create a class or method and call them when needed.
+1. While styling any custom module, Do add the styling for that specific module instead of adding it in the design theme so the style will not get loaded unless the module is being used. For example, like in app/code/Company/Module/view/frontend/web/css/source/_module.less.
```

Comment:
```suggestion
1. While styling any custom module, do add the styling for that specific module instead of adding it in the design theme so the style will not get loaded unless the module is being used. For example, like in app/code/Company/Module/view/frontend/web/css/source/_module.less.
```

Reply:
Thanks for the suggestion @rogyar. I guess I have fixed the error. Please review.